includes/Modules/Idea_Hub.php

```diff
@@ -339,6 +354,8 @@ public function is_connected() {
 			}
 		}
 
+		add_action( 'transition_post_status', $this->get_method_proxy( 'on_idea_hub_post_status_transition' ), 10, 3 );
+
```

Comment:
This needs to be defined in the `register` method withtin the `if ( $this->connected() )` condition.